[EDU-5388] Add E-commerce Astro Templates guide (#1301)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/astro-ecommerce-collection.mdx

---

> title: How to deploy an e-commerce with Astro using templates

Reviewer: ```suggestion
title: How to deploy an e-commerce based on Astro using templates
```

description: >-
  This guide contains instructions to create an e-commerce application using templates based on Astro.
meta_tags: >-
  Astro, e-commerce, templates, guides, Azion Marketplace, deployment, online store
namespace: docs_guides_astro_ecommerce_templates_collection
permalink: /documentation/products/guides/astro-ecommerce-collection/
---

import LinkButton from 'azion-webkit/linkbutton';
import Tag from 'primevue/tag';

<Tag severity="info" client:only="vue">
Preview
</Tag>

The **Astro E-commerce Templates** collection offers configurations for building e-commerce applications using templates. These templates are all based on the **Astro** framework.

The available templates are:

> - **Astro Audiophile**: create a stylish e-commerce based on Astro. Uses version `3.1.2`of Astro.
> - **Astro Ecommerce**: deploy a complete e-commerce. Uses version `2.0.1` of Astro.
> - **Astro Quickstore**: create your online store with Astro and Tailwind CSS. Uses versions `2.0.1` of Astro and `4.0.0-alpha.23` of Tailwind CSS.

Reviewer: ```suggestion
- **Astro Audiophile**: create a stylish e-commerce based on Astro. It uses version `3.1.2`of Astro.
- **Astro Ecommerce**: deploy a complete e-commerce. It uses version `2.0.1` of Astro.
- **Astro Quickstore**: create your online store with Astro and Tailwind CSS. It uses versions `2.0.1` of Astro and `4.0.0-alpha.23` of Tailwind CSS.
```

The deployment process for each of these templates creates a GitHub repository for your project, along with an edge application and a domain. This setup enables easy access and management of your e-commerce application through the Azion Edge Platform.

---

## Requirements

- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
    - Every push will be deployed automatically to this repository to keep your project updated.
- These templates use [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information.

---

## Deploying the template

You can obtain and configure your templates through the Azion Console. To easily deploy any of them at the edge, click the respective button below.

<LinkButton

>     label="Astro Audiophile"

Reviewer: ```suggestion
    label="Deploy Astro Audiophile"
```

    link="https://console.azion.com/create/azion-community/astro-audiophile"
    icon="ai ai-azion"
    icon-pos="left"
  />
<LinkButton

>     label="Astro Ecommerce"

Reviewer: ```suggestion
    label="Deploy Astro Ecommerce"
```

    link="https://console.azion.com/create/azion-community/astro-ecommerce"
    icon="ai ai-azion"
    icon-pos="left"
  />
<LinkButton

>     label="Astro Quickstore"

Reviewer: ```suggestion
    label="Deploy Astro Quickstore"
```
